core/domain/exp_services.py

```diff
@@ -561,6 +561,14 @@ def apply_change_list(exploration_id, change_list):
                         'version %s, received %s' % (
                             feconf.CURRENT_STATE_SCHEMA_VERSION,
                             change.to_version))
+
+        exp_android_proto = exploration.to_proto(
+            exploration.id, exploration.title, exploration.version,
+            exploration.init_state_name, exploration.states)
+        exp_android_proto_size = 0
+        if exp_android_proto != {}:
+            exp_android_proto_size = int(exp_android_proto.ByteSize())
+        exploration.update_proto_size_in_bytes(exp_android_proto_size)
```

Comment:
What could be the clean approach we can take here so that we can calculate proto size everytime the exploration get updated?

Reply:
It might be cleaner to  compute this in update_exploration(), instead, but I defer to others that are more familiar with the exploration pipeline.

Reply:
Yup, update_exploration() seems sensible -- it should be the last thing you do before the save() call.

Reply:
Done